src/main/java/io/github/fabriccommunity/everything/annotations/BestInGame.java

```diff
@@ -0,0 +1,17 @@
+package io.github.fabriccommunity.everything.annotations;
+
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.*;
+
+/**
+ * Target is best in game.
+ */
+@Target({FIELD,METHOD,PARAMETER})
```

Comment:
Field, method and parameter. What about package target? 👎until it can be used on packages.

Reply:
My latest commit should prove to be satisfactory, if not more so.